src/components/SessionList.vue

```diff
@@ -23,7 +23,7 @@
 <template>
 	<Popover class="session-list" placement="top">
 		<button slot="trigger"
-			v-tooltip.bottom="t('text', 'Participants')"
+			v-tooltip.bottom="t('text', 'Active people')"
```

Comment:
I'd even just say "People" for simplicity, also cause "Active" might not be true if they are afk for a few minutes?

Reply:
No strong opinion there, my initial thought was that it would make it a bit more obvious on what people you actually see there.

Reply:
Ok then let’s just go with it like this, we can always adjust :)